### PR TITLE
Add classes

### PR DESCRIPTION
**Pull request by Gbengacode**
### I Made the following changes to this work

- I added person class
- I added student class 
- I added teacher class

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom)
+    @classroom = classroom
+    super()
```
Comment (er-contreras) [inline student.rb]: - [ ] Don't forget to remove the super parenthesis, with it, it throws an error "wrong number of arguments". Calling super() calls the superclass (or ancestors) method without any parameter (assuming this method accepts no parameters...)

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
+    @specialization = specialization
```
Comment (er-contreras) [inline teacher.rb]: - [ ] Don't forget to remove the super parenthesis, with it, it throws an error "wrong number of arguments". Calling super() calls the superclass (or ancestors) method without any parameter (assuming this method accepts no parameters...)

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @Gbengacode, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
 - [ ] Set the attr_accessor at the beginning of the class that tells us since there which setters and getters we are managing. It would be well for the people who read your code to be aware of that.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi @Gbengacode 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [ ] N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!:clap::clap::clap:

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**